notebooks/work_in_progress/test_pudl_allocate_net_gen.ipynb

```diff
@@ -0,0 +1,724 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Do we have any policy on when we want to check in notebooks in the WIP folder to the main branch? I think ideally we'd move notebooks we want to keep out of WIP and into manual data or another folder when a PR is ready to merge into main

Reply:
I was planning on deleting this workbook before merging the PR. But in general, I think that we should have a policy that any notebook in WIP has to be associated with an active issue/PR, otherwise it needs to be deleted or moved to another directory.

Maybe though I should consider keeping this notebook since there are still some plants where the allocation isn't matching up completely.